devicetypes/smartthings/zwave-water-sensor.src/zwave-water-sensor.groovy

```diff
@@ -58,12 +58,12 @@ metadata {
 }
 
 def initialize() {
-	if (zwaveInfo.mfr.equals("0086"))
-		// 8 hour (+ 2 minutes) ping for Aeotec
+	if (zwaveInfo.mfr.equals("0086") || zwaveInfo.mfr.equals("021F") || zwaveInfo.mfr.equals("0258"))
```

Comment:
Let's create `is<mfr name>` or `is<mfr name><model name>` functions for these.

Reply:
Sure, done